flickit-assessment-data/src/main/java/org/flickit/assessment/data/jpa/kit/attribute/AttributeJpaEntity.java

```diff
@@ -0,0 +1,51 @@
+package org.flickit.assessment.data.jpa.kit.attribute;
+
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "baseinfo_qualityattribute", uniqueConstraints = {
+    @UniqueConstraint(name = "baseinfo_qualityattribut_title_assessment_subject_4a82494c_uniq", columnNames = {"title", "subjectId"}),
```

Comment:
In a Spring Boot project that uses Liquibase for database management, it is not necessary to explicitly define constraints in your data model classes (JpaEntity classes) if you have already defined them in Liquibase.

Liquibase is responsible for managing the database structure and handling database schema changes, including the creation of tables, columns, indexes, and constraints. By defining constraints in Liquibase changesets, you have already ensured that the constraints are applied consistently to the database.

In this scenario, your data model classes should reflect the structure and relationships of your domain entities without duplicating the constraints. The primary focus of these classes should be on representing the business logic and mapping the relationships between entities.

By relying on Liquibase for constraint management, you can avoid redundancy and maintain a single source of truth for your database schema. This approach also enables you to version and track changes to the database structure more effectively.

However, it's worth mentioning that some developers prefer to include constraints in both Liquibase and their data model classes for documentation purposes or to support database-independent testing. If you choose to follow this approach, ensure that the constraints are consistent between Liquibase and the data model classes to avoid conflicts or inconsistencies in the application behavior.

Ultimately, the decision to include constraints in your data model classes in addition to Liquibase depends on your specific project requirements and development practices

من شخصا وزن دردسرهای ناشی از حفظ consistency خیلی بیشتره برام و همیشه روشی رو انتخاب می‌کنم که از این دردسرها پیشگیری کنم. 
اون اهداف داکیومنتیشن و راحتی دولوپرها رو هم موافقم که اونجوری راحتتره ولی واقعا خوندن جنج‌ست‌های لیکوئیبیس هم خیلی راحته. یعنی خیلی راحت اسم table رو سرچ می‌کنیم تو پروژه و هیستوری تغییراتی که داشته و آخرین وضعیتش رو میشه از چنج‌ست‌ها دید.

ولی خب اگر تصمیم تیم بر این باشه که این تکرار رو انجام بدیم و زحمت حفظ consistency رو به جون بخریم، به نظرم باید یه جا برای همه انتیتی‌ها اعمال کنیم. اینکه فقط همین یه دونه الان اینجوری مرج بشه، احتمال اینکه حواسمون نباشه با تغییر لیکوئیبیس باید بیایم اینجا رو هم عوض کنیم خیلی بیشتر میشه و سوتی میدیم حتما

Reply:
موافقم، فقط برای تست نویسی integration به نظرم بهتر باشه تنظیم کنیم روی h2 تست‌ها رو و نیازی نداشته باشیم که حتما postgres بیاریم بالا.